src/libostree/ostree-repo.c

```diff
@@ -5099,3 +5702,47 @@ ostree_repo_set_collection_id (OstreeRepo   *self,
 
   return TRUE;
 }
+
+/**
+ * ostree_repo_get_lock_timeout:
+ * @self: an #OstreeRepo
+ *
+ * Get the current lock timeout for the repository. A timeout of -1 indicates
+ * that the application will block until the lock is acquired. Otherwise, it
+ * represents the number of seconds the application will sleep attempting to
+ * acquire the repository lock.
+ *
+ * Returns: lock timeout for the repository
```

Comment:
Nitpick: maybe add `, in seconds` to this?

Reply:
Sure.

Reply:
Will fix after #1343.